dockerfiles/alpine_3.5_1.10.3

```diff
@@ -91,8 +91,8 @@ RUN set -x \
         ./configure; \
         make -j ; \
         cp .libs/libprofiler.so* /usr/local/lib;) \
-    && (GOPATH=/usr/src/go go get github.com/google/pprof; \
-        cp /usr/src/go/bin/pprof /usr/local/bin) \
+    && (GOPATH=/usr/src/go go get github.com/google/pprof && \
+        cp /usr/src/go/bin/pprof /usr/local/bin || true ) \
```

Comment:
Reason of the failure: https://github.com/google/pprof/issues/538

I don't think that it is good to silence the check and provide non-working pprof. It is better to find a way to freeze older pprof version, which will support old go version we have here. Or remove pprof completely from old images and revive in the newer (where go version hopefully will be newer).

The check is great: otherwise we would not know that we provide a broken tool in our images.

Reply:
Right, I've reverted the changes and set to use 'wget' tool instead of 'go'.

Reply:
> wget -O /usr/local/bin/pprof github.com/google/pprof

1. C'mon, html in /usr/local/bin.
2. Neither commit message, not PR description was updated (still describes the old attempt to fix it; does not mention the problem root (the link is above)).

Does `go get` allows to stick with a certain version of a package? What pprof version is suitable for the installed go version?

Reply:
Right, the issue could be because current docker image with alpine 3.5
version has 'go' tool of the unsupported version:
  go version go1.11.5 linux/amd64
Check the pprof github issue:
  https://github.com/google/pprof/issues/538
  "The master branch is Incompatible with go version < 1.13 #538"
But in error log there was 1.14 version and it looks strange.

Reply:
Alternative (I don't know what is better, but it is shorter): `GO111MODULE=on go get github.com/google/pprof@160c4290`.

Reply:
Right, it could be better solution, but I couldn't get it work:
```
/opt/tarantool/pprof # GO111MODULE=on go get github.com/google/pprof@160c4290
go: finding github.com/google/pprof 160c4290
go get: inconsistent versions:
```